### PR TITLE
Admin filter people by campaign

### PR DESCRIPTION
**Pull request by jparkrr**
Related to what was proposed in #343 

I added a dropdown to filter the admin person view by campaign. But now I'm wondering if it would be better to instead return the campaigns associated with each texter and offer client-side filtering (on campaign or name).

Or maybe we want both? This is what I originally thought, but perhaps the added complexity isn't worth it. Feel free to close this if so.

I went with a url query parameter for the filter state so the graphql query could obtain access to it without splitting the component.

### CODE REVIEW

File: src/server/api/campaign.js [hunk 1]
```diff
@@ -111,6 +111,9 @@ export const resolvers = {
       r.table('assignment')
         .getAll(campaign.id, { index: 'campaign_id' })
         .eqJoin('user_id', r.table('user'))('right')
+        .eqJoin('user_id', r.table('user_organization')).zip()
+        .filter({ organization_id: campaign.organization_id })
```
Comment (jparkrr) [inline src/server/api/campaign.js]: Do these two lines seem ok? I wanted to be able to access the `role` from `campaign.texters`

**Comment by jparkrr** [comment]
Or, if you have other ideas for simplifying this, let me know.

File: src/server/api/campaign.js (hunk 1 above)
Comment (schuyler1d) [reply src/server/api/campaign.js]: Can we convert this to a knex query?  I think it'll be more straightforward there.
Also, not sure it's relevant, but you might want to look at this:
https://github.com/MoveOnOrg/Spoke/blob/scaling-messages/src/server/api/user.js#L4
seems like if we are doing user queries by org with similar aims, maybe worth using here, too.

Comment (schuyler1d) [reply src/server/api/campaign.js]: Also, the join data won't actually be used, unless you leverage it in this method:
https://github.com/MoveOnOrg/Spoke/blob/scaling-messages/src/server/api/user.js#L39
i.e. something like `user.orgRoles ? getRoles(user.orgRoles[organizationId]) :  ...`

**Review by schuyler1d — DISMISSED** [review]
please tweak sql and make sure it's leveraged in api/user.js.
Beyond that, maybe the organization.texters can take a campaignId and then you can get membership with another join that's exposed from the user object -- otherwise, we're doing the texters query twice on the backend -- not a big deal, but not great either.

**Comment by jparkrr** [comment]
Thanks for the review @schuyler1d. Updated to not touch the campaign query. This way makes a lot more sense.

**Comment by shakalee14** [comment]
@jparkrr - can you dig into why tests are hanging?

**Review by shakalee14 — APPROVED** [review]

**Pull request merged**